src/utils/__tests__/prettier-utils.test.ts

```diff
@@ -0,0 +1,110 @@
+import { formatJsx, formatCss } from "@/utils";
+import { expect } from "vitest";
+
+describe("prettier-utils", () => {
```

Comment:
These are cool but feel a bit redundant IMO, since the prettier-utils is only a wrapper to prettier we're basically testing prettier

Reply:
I totally agree, it was mainly added to test functionality of if we change the methods or add additions of our own, to make sure we're not breaking the behavior intended.

Also, it can help us upgrading the library without unexpected stuff.

Reply:
agreed to follow if there's a need. If the snapshots won't be updated for long time then it's probably redundant test